p2p/discover/discfilter/discfilter_test.go

```diff
@@ -0,0 +1,27 @@
+package discfilter
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStaticBan(t *testing.T) {
+	Enable()
+	cleanupInterval = 1 * time.Second
+	en1, _ := enode.Parse(enode.ValidSchemes, "enode://969654fc30f10e45903ee2d107cfc165feaff6f9acbd2a9472120a3581f7c717ded76113aa116e35b8389dfcc9129ecfc85a409f6c46fc864ce077cb85316ac8@3.252.226.57:5050")
+	BanStatic(en1.ID())
+	time.Sleep(2 * time.Second)
```

Comment:
Seeing that in the other PR you significantly reduced the Sleep time - wondering if you could do this here too?